test-requirements.txt

```diff
@@ -7,3 +7,5 @@ paramiko
 pytest
 pyyaml
 wget
+# for the libvirt provider
+# libvirt-python
```

Comment:
Lets just include this. The overhead for have an extra library inside a tox env is less than requiring a developer to find all the places to change these sorts of things.

Reply:
Done